app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -541,4 +545,12 @@ class PatientSummaryScreenController @Inject constructor(
         .ofType<PatientSummaryLinkIdCompleted>()
         .map { Ui::hideLinkIdWithPatientView }
   }
+
+
+  private fun isBpEditable(bloodPressureMeasurement: BloodPressureMeasurement, bpEditableFor: Duration): Boolean {
+    val now = Instant.now(utcClock)
+    val editExpiresAt = bloodPressureMeasurement.createdAt.plus(bpEditableFor)
```

Comment:
Suggestion for a slightly better name: `cannotEditAfter`?

Reply:
I have refactored this code for better use of `Duration`.
Please take a look.

Reply:
Looks good, I am going to rename `durationSince` to `durationSinceBpCreated` since the former doesn't really tell us anything.